### PR TITLE
Measure energy usage instead of emissions

### LINKED ISSUE #1321: Report energy usage rather than emissions
Rather than report carbon emissions in the emissions table, we should report energy usage as this is a more consistent metric.

### PR DESCRIPTION
**Pull request by jess-farmer**
## Description of work

Fixes #1321 

## Testing Instructions

1. Verify that the emissions table has been replaced by energy usage, reporting kWh rather than CO2 emissions.
2.
3.

## Checklist

- [x] The title is of a format appropriate for a line in future release notes.
- [x] I have added the appropriate tags to the PR.
- [x] My PR represents one logical piece of work.
- [x] My PR fully fixes the issue linked. If new issues have been created, what are they?
- [x] I have added the appropriate tests to cover code that has been added.
- [x] I have updated the documentation in the relevant places to cover the changes.

### CODE REVIEW

**Review by RabiyaF — CHANGES_REQUESTED** [review]
It looks good :) and it makes sense to shift to from carbon intensity to power usage. Just had a few suggestions.

File: fitbenchmarking/results_processing/fitting_report.py [hunk 1]
```diff
@@ -94,10 +94,7 @@ def create_prob_group(result, support_pages_dir, options):
         str(result.func_evals) if result.func_evals else "not available"
     )
 
-    if np.isnan(result.emissions):
-        emission_disp = "N/A"
-    else:
-        emission_disp = f"{result.emissions:.4g} kg CO\u2082 eq"
+    energy_disp = "N/A" if np.isnan(result.energy) else f"{result.energy:.4g}"
```
[the remark this answers is not in the document]
Comment (jess-farmer) [reply fitbenchmarking/results_processing/fitting_report.py]: Good spot - I was intending to keep the units there just missed them off!

File: fitbenchmarking/core/fitting_benchmarking.py [hunk 2]
```diff
@@ -69,7 +69,7 @@ def __init__(self, options, data_dir, checkpointer, label="benchmark"):
         self.__grabbed_output = output_grabber.OutputGrabber(self._options)
         self.__emissions_tracker = None
         self.__logger_prefix = "    "
-        if "emissions" in options.table_type:
+        if "energy_usage" in options.table_type:
             self.__emissions_tracker = EmissionsTracker()
```
[the remark this answers is not in the document]
Comment (jess-farmer) [reply fitbenchmarking/core/fitting_benchmarking.py:73]: I think reducing the interval to 1 second is a good idea - I've run a few experiments today and it doesn't appear to have a major impact on runtime

**Review by RabiyaF — APPROVED** [review]
Everything works great 💯 Thank you for adding this to `fitbenchmarking`.

**Pull request merged**